cmd/celestia/rpc.go

```diff
@@ -440,12 +441,14 @@ func parseNamespace(param string) (namespace.ID, error) {
 			return nil, fmt.Errorf("error decoding namespace ID: %w", err)
 		}
 	}
-	// if the namespace ID is 8 bytes, add v0 share + namespace prefix and zero pad
-	if len(nID) == 8 {
+	// if the namespace ID is <= 10 bytes, add v0 share + namespace prefix and zero pad
+	if len(nID) <= appns.NamespaceVersionZeroIDSize {
 		nID, err = share.NewNamespaceV0(nID)
 		if err != nil {
 			return nil, err
 		}
+	} else if len(nID) < appns.NamespaceSize {
+		return nil, fmt.Errorf("passed namespace is too large")
```

Comment:
Looks like you meany `>` instead of `<`
```suggestion
	} else if len(nID) > appns.NamespaceSize {
		return nil, fmt.Errorf("passed namespace is too large")
```